verl/trainer/ppo/core_algos.py

```diff
@@ -458,18 +458,18 @@ def kl_penalty(logprob: torch.FloatTensor, ref_logprob: torch.FloatTensor, kl_pe
     Returns:
```

Comment:
please update the list of values in `docs/examples/config.rst`.  remove the old names in the doc since we do not want others to use them. 

also please search in the codebase for existing values used in scripts. for example:
- examples/reinforce_plus_plus_trainer/run_qwen2-7b_math_rf.sh 
- examples/reinforce_plus_plus_trainer/run_qwen2-7b_math_rf_baseline.sh